### ISSUE TITLE
Why procedural parts are not supported ?

### ISSUE THREAD

**Issue by aishwarya25252** [judicator/SimpleRepaint#2]
If there is a blocker or issue for supporting procedural parts I would like to take it up and contribute .

**Comment by judicator** [comment]
Well, SimpleRepaint does not make any magic on it's on, but relies on the power of mighty B9PartSwitch mod. And B9PS sadly does not support Procedural Parts.
If you could figure out any solution - I'd be more than happy to implement it.

**Comment by aishwarya25252** [comment]
Yeah , but I was going through the configs seems like the procedural tanks from procedural parts just work fine . I am looking into the procedural wings will update if I figure out a solution .
